docs/getting-started/server/guide.mdx

```diff
@@ -35,14 +35,20 @@ Before you start: make sure you’ve installed the recommended
 
 2.  Open a terminal and navigate to the root of the cloned repository.
 
-## Configure Git blame
+## Configure Git
 
-Configure Git to ignore the Prettier revision:
+1. Configure Git to ignore the Prettier revision:
 
 ```bash
 git config blame.ignoreRevsFile .git-blame-ignore-revs
 ```
 
+2. Set up the pre-commit `dotnet format` hook:
```

Comment:
Maybe we should mark it as optional and add a disclaimer that dotnet format requires a full build which takes almost a minute to build.

Reply:
good point, done